### PR TITLE
some bug fixes

### PR DESCRIPTION
**Pull request by MatthewVita**

### CODE REVIEW

**Comment by MatthewVita** [comment]
@robertdown please review :)

File: patient.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import barnum
 from util import *
 
-
```
Comment (robertdown) [inline patient.py]: The second blank line needs to stay, https://www.python.org/dev/peps/pep-0008/#blank-lines

File: patient.py [hunk 2]
```diff
@@ -117,15 +116,16 @@ def generate_title(gender=None):
...
+
+    return weighted_titles[r - 1 if r > 0 else 0]
 
 
 def generate_address():
     street = barnum.create_street()
     zip, city, state = barnum.create_city_state_zip()
     return street, city, state, zip
 
-
```
Comment (robertdown) [inline patient.py]: Keep blank line https://www.python.org/dev/peps/pep-0008/#blank-lines

**Review by robertdown — CHANGES_REQUESTED** [review]
Looks good except for the two comments

**Comment by MatthewVita** [comment]
@robertdown done!

**Comment by robertdown** [comment]
Thanks for the contribution @MatthewVita!

**Comment by MatthewVita** [comment]
Glad to help. This tool is super useful!

**Pull request merged**